iothub/device/src/Transport/HttpTransportHandler.cs

```diff
@@ -239,7 +239,7 @@ public override async Task<Message> ReceiveAsync(CancellationToken cancellationT
 
             byte[] byteContent = await responseMessage.Content.ReadAsByteArrayAsync().ConfigureAwait(false);
 
-            using var message = byteContent != null
```

Comment:
This method returns that message, so it should be disposed by the caller. Oopsie!